src/components/Footer/Footer.tsx

```diff
@@ -3,17 +3,34 @@ import "./Footer.css";
 import { ReactComponent as LinkedIn } from "../../assets/linkedin-icon.svg";
 import { ReactComponent as GitHub } from "../../assets/github-icon.svg";
 
-function Footer() {
+interface FootersProps {
+  setIsDarkThemeOff: React.Dispatch<React.SetStateAction<boolean>>;
+  isDarkThemeOff: boolean;
+}
+
+function Footer(props: FootersProps) {
   return (
-    <nav className="wrapper" id="footer" data-testid="footer-wrapper">
+    <nav
+      className={`wrapper wrapper-${props.isDarkThemeOff ? "dark" : "light"}`}
```

Comment:
I suggest to create utility function because here is a lot of repeatable code (14, 24, 31 strings)

Reply:
good idea! 
done